client/components/Pagination.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+const Pagination = ({ itemsPerPage, totalItems, currentPage, changePage }) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(totalItems / itemsPerPage); i++) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <ul className="pagination">
+      {
+        currentPage > 1 && <li key="prev" className="pagination-item pagination-item__prev" onClick={ () => changePage(currentPage - 1) } >
+          Previous
+        </li>
+      }
```

Comment:
This is a weird way of doing an inline if statement that I found online. Pretty useful for when we want to selectively render stuff.

Basically, if the first part of the statement is true, the second part (after the &&) will render. If it's false, it won't.